Sources/NIOOpenSSL/SSLPrivateKey.swift

```diff
@@ -43,7 +131,13 @@ public class OpenSSLPrivateKey {
         let key: UnsafeMutablePointer<EVP_PKEY>?
         switch format {
         case .pem:
-            key = PEM_read_PrivateKey(fileObject, nil, nil, nil)
+            // This annoying conditional binding is used to work around the fact that I cannot pass
+            // a variable to a function pointer argument.
+            if let userData = pemUserData {
+                key = PEM_read_PrivateKey(fileObject, nil, globalOpenSSLPassphraseCallback(buf:size:rwflag:u:), userData)
```

Comment:
this is the point where we should go `Unmanaged<...>` to `UnsafeMutableRawPointer`